spec/services/open_weather_map/filter_min_max_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+require 'webmock/rspec'
+require 'open_weather_map/filter_min_max'
+
+RSpec.describe OpenWeatherMap::FilterMinMax do
+  describe '#get_data' do
+    let(:response_body) do
+      # Replace this with a sample OpenWeatherMap API response for testing
+      {
```

Comment:
Would ideally load from file; however this works and the candidate took time to author specs!